prov/rxd/src/rxd.h

```diff
@@ -246,6 +246,9 @@ static inline uint32_t rxd_flags(uint64_t fi_flags)
 	return rxd_flags;
 }
 
+#define rxd_after_eq(a,b)	((long)((a) - (b)) >= 0)
+#define rxd_before(a,b)		((long)((a) - (b)) < 0)
```

Comment:
Can you move these to common code?

Reply:
Sure. Where? include/ofi_util.h?
Also, do you want me to include the set of calls (ie ofi_after, ofi_before, ofi_after_eq, ofi_before_eq) or just the two I used?

Reply:
ofi_util.h seems fine for now.  Just the 2 calls you used.

Reply:
updated